frontend/src/components/Button/Button.styled.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+export const StyledButton = styled.button`
+  padding: 15px;
+  height: 30px;
+  width: 100px;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  border-radius: 5px;
+  border: none;
```

Comment:
margin, padding, border  - raczej powinny być koło siebie, niezależnie od konwecji ( i raczej zaraz pod height).

Reply:
Hmm to już nie wiem, u mnie w CR było mówione żeby robić position, top,right, padding, margin, width, height i potem display a na końcu resztę. To jak to w końcu jest.

Reply:
Oki